src/NATS.Client.Core/Internal/SubscriptionManager.cs

```diff
@@ -47,15 +47,20 @@ public SubscriptionManager(NatsConnection connection, string inboxPrefix)
 
     internal InboxSubBuilder InboxSubBuilder { get; }
 
-    public async ValueTask SubscribeAsync(string subject, NatsSubOpts? opts, NatsSubBase sub, CancellationToken cancellationToken)
+    public async ValueTask SubscribeAsync(string subject, string? queueGroup, NatsSubOpts? opts, NatsSubBase sub, CancellationToken cancellationToken)
     {
         if (IsInboxSubject(subject))
         {
+            if (queueGroup != null)
+            {
+                throw new NatsException("Inbox subscriptions don't support queue groups");
```

Comment:
I don't believe this is a technical limitation, is it? It does seem uncommon but I don't think it's worth throwing an exception on.

I was just thinking the Request abstraction would never hit this case because the Inbox Subject is never exposed. But someone could come up with their own pattern that does use an inbox subject with a queue group

Reply:
Fixed.